nft/collection/delegate_process.go

```diff
@@ -3,79 +3,313 @@ package collection
 import (
 	"sync"
 
-	"github.com/pkg/errors"
+	"github.com/ProtoconNet/mitum-account-extension/extension"
 	"github.com/spikeekips/mitum-currency/currency"
+	"github.com/spikeekips/mitum/base"
+	"github.com/spikeekips/mitum/base/operation"
 	"github.com/spikeekips/mitum/base/state"
 	"github.com/spikeekips/mitum/util/valuehash"
 )
 
+var DelegateItemProcessorPool = sync.Pool{
+	New: func() interface{} {
+		return new(DelegateItemProcessor)
+	},
+}
+
 var DelegateProcessorPool = sync.Pool{
 	New: func() interface{} {
 		return new(DelegateProcessor)
 	},
 }
 
-func (Delegate) Process(
+func (op Delegate) Process(
```

Comment:
@wyuinche 
If the receiver is not used inside a method then you better hide it.